scripts/release-setup.sh

```diff
@@ -2,10 +2,10 @@
 
 # this script should only be called within github actions running goreleaser
 
-sudo apt-get update
-sudo apt-get install clang
-sudo apt-get install gcc-multilib g++-multilib
-sudo apt-get install gcc-mingw-w64-x86-64 g++-mingw-w64-x86-64
+sudo apt-get update -y
+sudo apt-get install clang -y
+sudo apt-get install gcc-multilib g++-multilib -y
+sudo apt-get install gcc-mingw-w64-x86-64 g++-mingw-w64-x86-64 -y
```

Comment:
Adding these so that I could build goreleaser locally via `act`. Otherwise the build was hanging when running the GitHub actions locally.

Reply:
Do we still need these or was this for testing purposes?

Reply:
It shouldn't have any affect on CI runs of goreleaser, but local testing via `act` will not work without these `-y` flags.

I like having them checked in to code so the goreleaser setup can be used for local testing as-is. If you're OK with keeping I'd like to just keep & merge.  If you prefer without then I can remove.